src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/rules/MediationSuccessfulRuleTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.rules;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCase;
+import uk.gov.hmcts.cmc.ccd.sample.data.SampleData;
+
+import java.util.List;
+
+public class MediationSuccessfulRuleTest {
```

Comment:
Please add tests with multiple documents to verify their order does not make a difference

Reply:
will do